src/components/Shared/ReceiveModal/ReceiveModal.tsx

```diff
@@ -162,6 +178,8 @@ const ReceiveModal: FC<Props> = ({ onClose }) => {
             </div>
           )}
 
+          {error && <div className="my-4 text-red-500">{error}</div>}
```

Comment:
Should this be an error component because it's used in the same way in other places?

Reply:
Good idea! :+1: